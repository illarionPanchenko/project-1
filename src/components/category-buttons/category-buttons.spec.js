import React from 'react';
import {Provider} from "react-redux";
import { shallow, configure  } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import { store } from '../../index';
import Category from "./category-buttons";

configure({ adapter: new Adapter() });

const enzymeWrapper = shallow(<Provider store={store}><Category /></Provider>);

describe('Category buttons',()=>{
    it('snapshot', () => {
        expect(enzymeWrapper).toMatchSnapshot();
    })
});