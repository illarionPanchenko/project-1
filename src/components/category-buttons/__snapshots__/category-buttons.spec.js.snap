// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Category buttons snapshot 1`] = `ShallowWrapper {}`;
